docs/docs/AWSPrivateLinkConnection.md

```diff
@@ -1,207 +0,0 @@
-# AWSPrivateLinkConnection
```

Comment:
FYI doc regeneration job carried some additional changes from previous sdk generation where doc-generation was not run - changes not related - but fine to include in any case